src/main/java/teammates/storage/api/FeedbackResponseMonitorDb.java

```diff
@@ -45,8 +45,8 @@ FeedbackResponseRecordAttributes makeAttributes(FeedbackResponseRecord entity) {
      * Gets a set of response records with chosen duration and interval.
      */
     public List<FeedbackResponseRecordAttributes> getResponseRecords(long duration, long interval) {
-        long currentTimeInSec = System.currentTimeMillis() / 1000;
-        long startTimeInSec = currentTimeInSec - duration;
+        long currentTimeInMilisec = System.currentTimeMillis();
```

Comment:
Do you think we don't need this variable, but instead just use `System.currentTimeMillis()` which is well named?